SoftServeCarMobile/src/activity/journey/journey-activity/segment-control-activities/journey-page/JourneyPage.tsx

```diff
@@ -1,56 +1,24 @@
-import { useNavigation } from "@react-navigation/native";
-import Moment from "moment";
 import React, { useEffect, useState } from "react";
-import { FlatList, Image, Text, TouchableOpacity, View } from "react-native";
-import { Divider } from "react-native-elements";
-import Ionicons from "react-native-vector-icons/Ionicons";
-import BottomSheet from "reanimated-bottom-sheet";
+import { Image, Text, TouchableOpacity, View, FlatList } from "react-native";
 import { container } from "tsyringe";
 import JourneyService from "../../../../../../api-service/journey-service/JourneyService";
-import { Journey } from "../../../../../../models/Journey";
 import { Stop } from "../../../../../../models/Stop";
 import { StopType } from "../../../../../../models/StopType";
 import { User } from "../../../../../../models/User";
 import BottomPopup from "../../../../../components/bottom-popup/BottomPopup";
-import MenuButton from "../../../../../components/bottom-popup/menu-button/MenuButton";
 import JourneyPageStyle from "./JourneyPageStyle";
+import { Journey } from "../../../../../../models/Journey";
+import { useNavigation } from "@react-navigation/native";
+import { LinearTextGradient } from "react-native-text-gradient";
+import { Divider } from "react-native-elements";
+import Moment from "moment";
+import Ionicons from "react-native-vector-icons/Ionicons";
 
-const JourneyPage = ({ route }: any) => {
-    const { journeyId } = route.params;
-
-    const [moreOptionsState, setMoreOptionsState] = useState(0);
+const JourneyPage = ({ props }: any) => {
```

Comment:
Ought to omit "any" type, since "any" may accept null or undefined which can be avoided using typing

Reply:
No, this code looks stable